collector/container/konflux.Dockerfile

```diff
@@ -75,21 +75,21 @@ ENV WITH_RHEL_RPMS=true
 
 # Build with gperftools (DISABLE_PROFILING=OFF) only for supported
 # architectures, at the moment x86_64 only
-RUN ./builder/install/install-dependencies.sh && \
-    if [[ "$(uname -m)" == "x86_64" ]];   \
+RUN ./builder/install/install-dependencies.sh
+RUN if [[ "$(uname -m)" == "x86_64" ]];   \
```

Comment:
Can you share why you split the RUN instruction into multiple? Debugging / Workspace / something else?

Reply:
Understood. Please add a comment explaining that, otherwise someone might 'optimize' the stages again. 
I think this is in the builder stage and we copy the resulting binaries into the final stage, so we don't have to optimize for layers here. 

https://github.com/stackrox/collector/pull/2027/files#diff-dd2e1150313169ab18110018f7c1e085ab8c31e164916b63883f5962afb49e79L149-L150